styles/home.css

```
@@ -0,0 +1,189 @@
+  }
+
+}
+
+@media (max-width: 743px) {
+  .banner {
+    height: 540px;
+  }
+
+  #hero .wrapper{
+
+  }
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)